token/cli/src/clap_app.rs

```diff
@@ -191,6 +191,7 @@ pub enum CliAuthorityType {
     Metadata,
     GroupPointer,
     GroupMemberPointer,
+    TokenGroup,
```

Comment:
nit: can you change this to just `Group`? We don't use the `Token` prefix anywhere else
```suggestion
    Group,
```

Reply:
My thought was to match the extension type, such as TokenMetadata and TokenGroup. But, I do see now that you use Metadata for the CliAuthorityType.